test/Microsoft.AspNetCore.Server.KestrelTests/LifetimeNotImplemented.cs

```diff
@@ -35,7 +35,6 @@ public CancellationToken ApplicationStopping
 
         public void StopApplication()
         {
-            throw new NotImplementedException();
```

Comment:
I think that's another leftover from when I was playing around with the code. Will revert and see what happens.